src/db/__init__.py

```diff
@@ -43,6 +43,26 @@ def get_next_survey_question(user_id, survey_id):
     return question, answers
 
 
+def get_survey_answer_id_by_alias(alias):
+    survey_answer_id = _fetchone(
+        f"SELECT survey_answer_id FROM survey_answer WHERE alias='{alias}'"
+    )
+    print("xxx " + str(survey_answer_id))
+    return _get_first_or_none(survey_answer_id)
+
+
+def check_if_survey_exists(survey_id):
+    survey = _fetchone(f"SELECT * from survey WHERE survey_id={survey_id}")
+    return _get_first_or_none(survey) is not None
+
+
+# User have previously answered on survey with answer which have this alias
+def find_all_by_alias(alias):
```

Comment:
Hm, this one is hard to name 😄 I'd rename this to `find_all_answer_alias_responders` or `find_all_answer_responders_by_alias`. The reason I don't like the name is that it's unclear what's actually happening. Find what? By what alias?

Reply:
Ok I choosed find_all_answer_alias_responders